normandy/conftest.py

```diff
@@ -22,3 +23,13 @@ def django_cache(request):
     yield None
     for cache_name in settings.CACHES.keys():
         caches[cache_name].clear()
+
+
+@pytest.fixture
+def geolocation():
+    """Fixture to load geolocation data."""
+    geolocation_module.load_geoip_database()
+    if not geolocation_module.has_geoip:
+        pytest.skip()
```

Comment:
I want a way to make skips turn into failures in CI. Sumo has this with nose, and it helps us be sure about our CI. I can't find a thing in pytest to do this.